Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:22.04
+ENV TZ="America/Chicago"
+RUN apt-get update
+RUN apt-get update && \
+    apt-get install -yq tzdata && \
+    ln -fs /usr/share/zoneinfo/America/Chicago /etc/localtime && \
+    dpkg-reconfigure -f noninteractive tzdata
+RUN apt-get upgrade -y
+RUN apt-get install -y vim magic-wormhole aptitude python3-pip docker.io
+COPY ./requirements.txt /root/requirements.txt
+RUN pip install -r /root/requirements.txt
+WORKDIR /root/
```

Comment:
I think this would be a fine thing to do, but I also think that the ubuntu:22.04 image this is based on is really slim at about 70-80 megabytes. It'll be a larger with the packages we add to it, but I also think that those are handy debugging tools when something is not working as expected inside the image. 

I took `aptitude` (ncurses package manager for ubuntu) and `docker.io` (docker client) out of the Dockerfile since they are from my new image boilerplate, and it builds to about a half-gig. Not amazing, but not terrible. I'm not sure really what the goal image size should be a for a production flask app. 🤔

Reply:
i'm not thinking so much about size, more about Python issuing ready-made docker images with less boilerplate and presumably their preferred configuration. Seems like the easiest way to be consistent and avoid any future confusion/issues when it comes to an upgrade path.  E.g., I can't tell from the dockerfile what version of Python is being installed?

Reply:
Thank you, Frank! Regardless of what base image we use, this has me thinking of all of our python-based images out there that probably need an upgrade plan.... 😬